buildcontext/git.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 const (
-	defaultGitImage = "alpine/git:v2.30.1"
+	defaultGitImage = "earthly/alpine-git-lfs:v2.36.3"
```

Comment:
How are we managing the release of this image? Does it go out together with other images like earthly/dind?

Reply:
that would be a reasonable place to handle that; so far it's just a one-off I made via

```
VERSION 0.7

gitlfs:
  ARG VERSION=v2.36.3
  FROM alpine/git:$VERSION
  RUN apk add git-lfs
  SAVE IMAGE --push earthly/alpine-git-lfs:$VERSION
```

but obviously needs to be stored somewhere besides a github comment.

Reply:
Oh BTW this image needs to be cross-platform such that is also works natively on arm64 too. Let's establish a proper release practice for this image before merging this PR.

Reply:
Also fully qualified so it works on podman

Reply:
this change ended up not being needed, as remotely referenced targets are fetched via the git binary in the buildkit container.